ownership-chain/runtime/src/lib.rs

```diff
@@ -334,7 +331,15 @@ impl pallet_authorship::Config for Runtime {
 }
 
 parameter_types! {
-	pub const ExistentialDeposit: Balance = EXISTENTIAL_DEPOSIT;
+/// The minimum amount required to keep an account open. MUST BE GREATER THAN ZERO!
```

Comment:
this comment seems unnecessary because we are setting it to 0 ourselves. Some explanation about why we are setting it to 0, how it could cause "replay attack" would be more useful than this

Reply:
comment should be indented to align with the code

Reply:
sgtm